app/templates/components/nav-bar.hbs

```diff
@@ -1,9 +1,20 @@
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet"/>
 <div class="mobile hidden row">
   <div class="ui menu">
     <LinkTo
       @route="index" class="header item" @activeClass="">
       <h3>{{this.settings.appName}}</h3>
     </LinkTo>
+    <div class="ui container" style="max-width: 400px;">
+      <div class="ui input" style="padding: 10px; margin-left: 160px; min-width: 900px;">
+        <button type="submit" style="color: white; border-color: white;"><i class="fa fa-search" style="color: lightgray;"></i></button>
```

Comment:
semantic ui icons are used

Reply:
What will be the better way rather than going with sematics?

Reply:
semantic ui icons will be the better way rather than going with sematics

Reply:
@iamareebjamal this is fine?
![Screenshot (222)](https://user-images.githubusercontent.com/47476857/102118026-23e8a400-3e65-11eb-9a7b-05ef03b2bb7b.png)